{"category": "ham", "to_address": "\"J.M. Maurer\" <uwog@uwog.net>", "from_address": "Marco Pesenti Gritti <mpg@redhat.com>", "subject": "Re: [sugar] Activity sharing broken", "body": "J.M. Maurer wrote:\n> Whenever I try to share an activity, I get an error about get_id() not\n> being defined on an Activity (and if you add it just for the heck of it,\n> it complains about get_service_name missing). For example:\n>\n> DEBUG - root: ActivityService.set_active: 1.\n> DEBUG - root: Requesting share of activity\n> 70ffd154d3b749edde51c3800ce9f581bd7d4032.\n> STDERR - Traceback (most recent call last):\n>   File\n> \"/Development/sugar-jhbuild/build/lib/python2.4/site-packages/sugar/activity/activity.py\", line 79, in _share_clicked_cb\n>     self._activity.share()\n>   File\n> \"/Development/sugar-jhbuild/build/lib/python2.4/site-packages/sugar/activity/activity.py\", line 313, in share\n>     self._pservice.share_activity(self)\n>   File\n> \"/Development/sugar-jhbuild/build/lib/python2.4/site-packages/sugar/presence/presenceservice.py\", line 377, in share_activity\n>     actid = activity.get_id()\n> AttributeError: 'AbiWordActivity' object has no attribute 'get_id'\n>   \n\nShould be fixed (just update sugar). Sorry, I didn't realize this was \nused by Activity subclasses.\n\nMarco\n_______________________________________________\nSugar mailing list\nSugar@lists.laptop.org\nhttp://lists.laptop.org/listinfo/sugar\n\n"}